cmd/state.go

```diff
@@ -40,7 +40,7 @@ func newStateCmd() *cobra.Command {
 		Short: "Edit the current stack's state",
 		Long: `Edit the current stack's state
 
-Subcommands of this command can be used to surgically edit parts of a stack's state. These can be useful when
+Subcommands of this command can be used to surgically edit parts of a stack's deployment state. These can be useful when
```

Comment:
This doesn't seem to be a net improvement, and I don't think is consistent with the typical way we refer to "state".

Reply:
I'm OK with not making this change.